Update config to include colors

paolorau opened the pull request:
## Purpose

- Update the component to use provided colors

## Asana tasks

- [Update background colors in AssetPicker](https://app.asana.com/0/0/1203135611977724/f)

## Approach

- Created a new struct `AssetsPickerColors` that hold the different colors used by the component, it also provides default values (using semantic values for iOS 13+)
- Replace the old colors.

## Screenshots

- 

https://user-images.githubusercontent.com/2923778/195395432-77e0248d-dfce-4a3a-8f3d-52126df01d6f.MP4

paolorau committed:
create class to hold new color values, update usage

paolorau committed:
update doc

Jmats17 reviewed (approved):
LGTM!

alanipce commented on AssetsPickerViewController/Classes/Picker/AssetsPickerColors.swift:
```diff
@@ -0,0 +1,64 @@
+    self.cellBackground = cellBackground ?? defaultColors.cellBackground
+  }
+  
+  struct defaultColors {
```
Use pascal case for type definitions

alanipce commented on AssetsPickerViewController/Classes/Picker/AssetsPickerColors.swift:
```diff
@@ -0,0 +1,64 @@
+
+/// Class to hold color values for Picker elements.
+public struct AssetsPickerColors {
+  var defaultCheckmark: UIColor
```
is the default checkmark used?

paolorau replied:
Good catch, I forgot to move `SSCheckMark.swift`

alanipce commented on AssetsPickerViewController/Classes/Album/Controller/AssetsAlbumViewController.swift:
```diff
@@ -94,7 +94,7 @@ open class AssetsAlbumViewController: UIViewController {
         super.loadView()
         view = UIView()
-		view.backgroundColor = .ap_background
+        view.backgroundColor = AssetsManager.shared.pickerConfig.colors.background
```
can't you use the `pickerConfig` instance defined on the album controller

paolorau replied:
I've removed the usage of the shared object in the places where I have access to the picker config. 
@alanipce Should I take some time to remove all the usages? I.e for the cells I can quickly create a `configure` method and pass the proper data, please let me know.

alanipce reviewed (commented):
Left some general comments
- Would be ideal if we could avoid accessing the shared config to access the color definitions, on the controller we can access the config directly
- The defaultColors type is in camel case
- Work with Dmitry on updating the repo specs and using it on the Albert app

paolorau committed:
Addressing comments

use defaultCheckmark, update struct name, use picker configuration directly when possible

paolorau committed:
replace F0F0F0 with cellBackground
